guide-redaction/index.md

```diff
@@ -3107,13 +3107,13 @@ title: "Guide de rédaction du contenu du site Canada.ca"
     <li><a href="https://www.nngroup.com/topic/writing-web/">Writing for the Web</a> (en anglais seulement), Nielsen Norman Group</li>
   </ul>
 </section>
-<aside class="wb-fnote wb-init wb-fnote-inited" role="note" id="wb-auto-4">
+<aside class="wb-fnote" role="note">
 	<h3 id="fn">Notes de bas de page</h3>
 	<dl>
-		<dt id="fn1-dt">Footnote 1</dt>
+		<dt>Footnote 1</dt>
 		<dd id="fn1" tabindex="-1">
-			<p>International Plain Language Federation: <a href="​​https://www.iplfederation.org/plain-language">​https://www.iplfederation.org/plain-language</a></p>
```

Comment:
I'm tempted to add _(en anglais seulement)_ but they have that one paragraph in French.

Reply:
Yeah, I had that in my original comments on the working doc I think , but Chelsey was OK with it. We can double check and make another edit if needed

Reply:
No need. If Chelsey already approved it without it. It's a grey area.